Orange/data/table/impl.py

```diff
@@ -0,0 +1,427 @@
+import warnings
+
+from Orange.data import ContinuousVariable, Domain, StringVariable, DiscreteVariable
+
+from .base import *
+import pandas as pd
+from pandas.sparse.array import BlockIndex
+
+
+class Table(TableBase, pd.DataFrame):
+    """A dense implementation of an Orange table."""
+
+    _KNOWN_PANDAS_KWARGS = {"data", "index", "columns", "dtype", "copy"}
```

Comment:
These can easily go out of sync. Best go [the other way around](https://github.com/biolab/orange3/pull/1347/#discussion_r74277114).